python-client/pyproject.toml

```diff
@@ -71,8 +72,9 @@ test = [
     "tensorflow-macos>=2.10.0, <2.11; sys_platform == 'darwin' and platform_machine == 'arm64'",
     "tensorflow>=2.10.0, <2.11; sys_platform != 'darwin' or platform_machine != 'arm64'",
     "tensorflow-text>=2.10.0, <2.11; platform_machine == 'x86_64' or platform_machine == 'amd64'",
-    "mlflow>2",
-    "wandb"
+    "mlflow>2",                                                                                    # Should it be somewhere else ?
+    "wandb",                                                                                       # Should it be somewhere else ?
```

Comment:
no it's an optional dependency that we'll ask users to install if they try to use W&B integration.
We install it for tests because there are unit tests for it

Reply:
I will remove comment then